tests/excmds.test.ts

```diff
@@ -6,6 +6,56 @@ const Options = require("selenium-webdriver/firefox").Options
 
 jest.setTimeout(60000)
 
+// API docs because I waste too much time looking for them every time I go back to this:
+// https://seleniumhq.github.io/selenium/docs/api/javascript/
+
+const vimToSelenium = {
+        "Down": webdriver.Key.ARROW_DOWN,
+        "Left": webdriver.Key.ARROW_LEFT,
+        "Right": webdriver.Key.ARROW_RIGHT,
+        "Up": webdriver.Key.ARROW_UP,
+        "BS": webdriver.Key.BACK_SPACE,
+        "Del": webdriver.Key.DELETE,
+        "End": webdriver.Key.END,
+        "CR": webdriver.Key.ENTER,
+        "Esc": webdriver.Key.ESCAPE,
+        "Home": webdriver.Key.HOME,
+        "PageDown": webdriver.Key.PAGE_DOWN,
+        "PageUp": webdriver.Key.PAGE_UP,
+        "Tab": webdriver.Key.TAB,
+        "lt": "<",
+}
+
+const modToSelenium = {
+        "A": webdriver.Key.ALT,
+        "C": webdriver.Key.CONTROL,
+        "M": webdriver.Key.META,
+        "S": webdriver.Key.SHIFT,
+}
+
+function translateKeys(string) {
+        return string.replace(/<([^>-]+)-?([^>]*)>/g, (wholematch, modifier, key, pos, str) => {
+                // There only is a modifier if 'key' isn't empty (<Up> = no modifier)
+                if (key.length > 0) {
+                        return webdriver.Key.chord(
+                                ...(modifier.split("").map(m => modToSelenium[m])),
+                                (vimToSelenium[key] || key),
+                                webdriver.Key.NULL,
+                        )
+                }
+                return vimToSelenium[modifier] || modifier
+        })
+}
+
+function sendKeys (driver, keys) {
+        return translateKeys(keys).split("")
+                .reduce((prom, key) => {
+                        return prom.then(_ => driver.wait(driver.switchTo().activeElement()))
+                                .then(elem => elem.sendKeys(key))
+                                .then(_ => driver.sleep(5))
```

Comment:
FWIW, fastest typists in the world type at about 200 WPM peak, so about 1000 CPM, which equates to a 60ms interval between keys. You can therefore safely increase this to 50ms, unless you're concerned about macros.

Reply:
I'd rather not make the tests slower that they need to be, running a single test already takes about 7 seconds on my machine :(